mvn/archetypes/radix/src/main/resources/archetype-resources/workflow/src/main/resources/bin/wmgr

```diff
@@ -86,7 +92,7 @@ if [ "$1" = "start" ]; then
     -Djava.io.tmpdir="$OODT_TMPDIR" \
     -Dorg.apache.oodt.cas.pge.task.metkeys.legacyMode="true" \
     -Dorg.apache.oodt.cas.pge.task.status.legacyMode="true" \
-    org.apache.oodt.cas.workflow.system.WorkflowManagerStarter \
+    org.apache.oodt.cas.workflow.system.XmlRpcWorkflowManager \
```

Comment:
Let's keep `WorkflowManagerStarter` as it is since it determine which workflow manager to start.

Reply:
resolved 👍